js/views/SurveyListPage.js

```diff
@@ -54,7 +54,7 @@ const SurveyListPage = React.createClass ({
   componentDidMount() {
     this.mountTimeStamp = Date.now()
 
-    // Update Survey List from Parse only once every 3 minutes
+    // Automatically update Survey List from Parse only once every 3 minutes
```

Comment:
This has been removed in master due to the overhead of reloading all the associated forms, questions, triggers.  Now that pull-to-refresh has been implemented, I don't see a need to automatically fetch data.  Let the user determine when they want to refresh so they can expect the network/processing delay as typical in a REST application